packages/eslint-plugin-pf-codemods/test/rules/v5/table-rename-TableComposable.js

```diff
@@ -17,7 +17,7 @@ ruleTester.run("table-rename-TableComposable", rule, {
   invalid: [
     {
       code: `import { TableComposable } from '@patternfly/react-table'; <TableComposable>Body</TableComposable>`,
-      output: `import { Table } from '@patternfly/react-table'; <Table data-codemods="true">Body</Table>`,
+      output: `import { Table /* data-codemods */ } from '@patternfly/react-table'; <Table>Body</Table>`,
```

Comment:
Is the `/* data-codemods */` in the output here the intended behavior?

Reply:
Yeah, I updated how we track what imports to not update paths for with the comment (instead of the data attribute).

Reply:
would we want to note this in the readme doc?